### PR TITLE
test(NODE-5238): set min/max range options when required.

### PR DESCRIPTION
**Pull request by durran**
Patch against this exact commit: https://spruce.mongodb.com/version/6451168856234306eb1923f7/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

### CODE REVIEW

File: bindings/node/test/clientEncryption.test.js [hunk 1]
```diff
@@ -763,6 +765,8 @@ describe('ClientEncryption', function () {
         queryType: 'rangePreview',
         contentionFactor: 0,
         rangeOptions: {
+          min: new BSON.Int32(0),
+          max: new BSON.Int32(10),
```
Comment (addaleax) [inline bindings/node/test/clientEncryption.test.js:769]: I guess index.d.ts also needs an update?

Comment (durran) [reply bindings/node/test/clientEncryption.test.js:769]: Tests just started failing with https://github.com/mongodb/libmongocrypt/commit/aa145adb538aeaf530e530ee801ef81391e655b4 so needed to update.

Comment (addaleax) [reply bindings/node/test/clientEncryption.test.js:769]: The types indicate that `min` and `max` are optional, though – is that still accurate? (It's fine if it is, I just assumed that it isn't anymore based on the PR title)

Comment (durran) [reply bindings/node/test/clientEncryption.test.js:769]: Ah... Good catch. They are required now.

**Review by W-A-James — APPROVED** [review]

**Review by addaleax — APPROVED** [review]

**Review by nbbeeken — APPROVED** [review]

**Pull request merged**